packages/db/src/lib/retry.ts

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+import pRetry from 'p-retry';
+import Logger from '@cumulus/logger';
+
+type PromiseReturnType<T> = T extends (method: any) => infer R ? Promise<R> : never;
```

Comment:
not sure about the type definition here.
Should `method: any` be `method: function`?

Reply:
https://github.com/nasa/cumulus/pull/3332/commits/0962c4bd57ba8cc70c4df475c8fed21f1c06639e